documentation/library_guide/api_for_sycl_kernels/tested_standard_cpp_api.rst

```diff
@@ -450,15 +450,17 @@ C++ Standard API                      libstdc++  libc++     MSVC
 ``std::truncf``                       Tested     Tested     Tested
 ===================================== ========== ========== ==========
 
-These tests were done for the following versions of the standard C++ library:
+These tests were done for the following SYCL compilers and versions of the standard C++ library:
```

Comment:
As far as I understand, a compiler served as a tool for testing a standard library. E.g. an additional, AdaptiveCPP, was used to test libc++, because Intel(R) oneAPI DPC++/C++ Compiler is linked against libstdc++. 

The suggestion shifts focus to the libraries:

```suggestion
The testing was conducted for the following versions of the C++ standard libraries, with the help of the listed compilers.:
```

It may also make sense to put the compiler into the last column and name it as "SYCL Compiler Used".

Reply:
Done.